experiments/tf_trainer/common/model_runner.py

```diff
@@ -51,7 +51,14 @@ def __init__(self, dataset: ds.DatasetInput,
     self._model = model
     self._estimator = model.estimator(self._model_dir())
 
+  # TODO(ldixon): consider early stopping. Currently steps is  hard coded.
   def train_with_eval(self, steps, eval_period, eval_steps):
+    """
```

Comment:
Same as above, I think the style guide requires some text after the initial quotes. Maybe:

"""Trains model.

Args:
...
"""